hack/lib/test.sh

```diff
@@ -126,12 +128,12 @@ kube::test::get_object_jsonpath_assert() {
 kube::test::describe_object_assert() {
   local resource=$1
   local object=$2
-  local matches=${@:3}
+  local matches=( "${@:3}" )
```

Comment:
Fix for this:

```
In ./hack/lib/test.sh line 131:
  result=$(eval kubectl describe "${kube_flags[@]}" ${resource} ${object})
                                                    ^---------^ SC2086: Double quote to prevent globbing and word splitting.
                                                                ^-------^ SC2086: Double quote to prevent globbing and word splitting.
```

Reply:
this comment doesn't match 😅

Reply:
Derp. Looked at new line number instead of existing line number. This is the correct spellcheck message for this finding:

```
In ./hack/lib/test.sh line 129:
  local matches=${@:3}
                ^----^ SC2124: Assigning an array to a string! Assign as array, or use * instead of @ to concatenate.
```